homu/server.py

```diff
@@ -29,6 +29,10 @@ def index():
 
 @get('/queue/<repo_label>')
 def queue(repo_label):
+    logger = g.logger.getChild('queue')
```

Comment:
Hmm, then, that would be OK!

Alas, the function call itself has a huge overhead in Python... Someone showed me that just merging all the functions into one function made the code faster. I just have to stop micro-optimizing something when using Python. :/